tests/target/string-lit.rs

```diff
@@ -25,5 +25,10 @@ not
 remove
 formatting"#;
 
+    let xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx = funktion("yyyyyyyyyyyyyyyyyyyyy\
+                                                                            yyyyyyyyyyyyyyyyyyyyy\
+                                                                            yyyyyyyyyyyyyyyyyyyyy\
+                                                                            yyyyyyyyyy");
+
```

Comment:
It's not an improvement on the test source, but it *is* an improvement on how rustfmt would previously format this. It would put everything on a single line.